tests/test_preprocessing.py

```diff
@@ -310,6 +312,7 @@ def test_scale_matrix_types(array_type, zero_center, max_value):
         sc.pp.scale(adata_casted, zero_center=zero_center, max_value=max_value)
     X = adata_casted.X
     if "dask" in array_type.__name__:
+        assert not isinstance(X._meta, np.matrix)
```

Comment:
I verified that this catches the issue without my fix

Reply:
so this means it's ok to ignore the coverage warning right?